py/desispec/scripts/link_calibnight.py

```diff
@@ -0,0 +1,138 @@
+
+import os, sys
+import argparse
+
+from desiutil.log import get_logger
+
+from desispec.io import findfile
+from desispec.io.util import decode_camword, relsymlink
+
+def parse(options=None):
+    """parse options from sys.argv or input list of options"""
+    p = argparse.ArgumentParser()
+    p.add_argument('--refnight', type=int, required=True,
+                   help='Reference night with calibs')
+    p.add_argument('--newnight', type=int, required=True,
+                   help='New night without calibs, to link to refnight')
+    p.add_argument('-c', '--cameras', type=str, default='a0123456789',
+                   help='Camword of cameras to link, e.g. a0123b6r7z8')
+    p.add_argument('--include', type=str, 
```

Comment:
Can you update this and exclude to mention that the code expects a comma-separated list without spaces?

Reply:
done: help string says comma separated, and splitting code also strips spaces now.